### PR TITLE
Backport foreman-obsolete-packages to 3.4

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+%prep
+
+%build
+
+%install
+
+%files
+
+%changelog
+* Fri Jan 27 2023 Evgeni Golov - 1.1-0.1
```
Comment (ekohl) [inline packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec:24]: This kind of makes me wonder if we need to introduce foremandist in the release. Now the changelog goes back in version numbers, which isn't great. On the other hand, foremandist wouldn't solve that either.

Perhaps we should just drop the changelog and introduce it as version 0.1.

Comment (evgeni) [reply packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec:24]: I thought about 0.9, but yeah, anything below 1.0 would work.

Really, don't mind any specific version number, as long as we can smoothen the 3.3→3.4 upgrade.

Comment (ekohl) [reply packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec:24]: Let's go with 0.9 then.

Comment (evgeni) [reply packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec]: Like this?

**Review by ekohl — APPROVED** [review]

**Comment by evgeni** [comment]
```console
% koji add-pkg --owner kojiadm foreman-3.4-el8 foreman-obsolete-packages
Adding 1 packages to tag foreman-3.4-el8
```

**Pull request merged**